osmaxx/conversion/converters/detail_levels.py

```diff
@@ -19,7 +19,7 @@
         included_layers=[
             'adminarea_a', 'boundary_l', 'geoname_l', 'geoname_p', 'landuse_a', 'military_a', 'military_p', 'misc_l',
             'natural_a', 'natural_p', 'poi_p', 'pow_p', 'railway_l', 'road_l', 'route_l', 'utility_p', 'water_a',
-            'water_p', 'water_l', 'transport_l', 'coastline_l', 'landmass_a', 'ocean_a',
+            'water_p', 'water_l', 'transport_l', 'coastline_l', 'landmass_a', 'sea_a',
         ],
         level_folder_name='level-{}'.format(str(DETAIL_LEVEL_REDUCED)),
     ),
```

Comment:
To avoid future inconsistencies, consider to
```python
assert set(DETAIL_LEVEL_TABLES[DETAIL_LEVEL_REDUCED]).issubset(DETAIL_LEVEL_TABLES[DETAIL_LEVEL_ALL])
```
either in a test or directly in `osmaxx/conversion/converters/detail_levels.py`.